wren-ui/src/apollo/server/repositories/relationshipRepository.ts

```diff
@@ -178,4 +179,25 @@ export class RelationRepository
     );
     return result.map((r) => this.transformFromDBData(r)) as RelationInfo[];
   }
+
+  public async findDuplicateRelationBetweenModels(modelIds) {
```

Comment:
nit: maybe rename to `findExistedRelationshipBetweenModels`

Reply:
delivered in 511d5faf675f40bd516a00d5258178530bae5d5b